app/src/main/AndroidManifest.xml

```diff
@@ -46,13 +46,12 @@
         </activity>
         <activity
             android:name="com.github.pockethub.ui.gist.CreateGistActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize">
+            android:configChanges="orientation|keyboardHidden|screenSize"
+            android:label="@string/create_gist">
```

Comment:
The label in the manifest is not only used in the activity itself, but also to label the activity in intent-choosers